### DISCUSSION TITLE
Shortcut to open a Catalog Item in Portal

### DISCUSSION THREAD

**Discussion by krysbzh** [arnoudkooi/SN-Utils#423] [Ideas]
Hi guys,
Regularly, I configure catalog items and I want to test them on my portal. There is the "Try it" button but which refers to the backoffice view. I have to copy/paste the sys_id and paste it into the url "/sp?id=sc_cat_item&sys_id=XXXX"
Could we not add a feature in SN Utils to directly open the item from the cat_item form.
The ideal would be to be able to select the portal (like the "Try it" button in the Page form). For now, I created a shortcut "/ci" which redirects me to /sp?id=sc_cat_item&sys_id=$sysid .

Any other ideas?
Chris

**Comment by GirlGeek98** [comment]
Hey - there is a Share project that will add this as a button to your catalog items.  https://developer.servicenow.com/connect.do#!/share/contents/3610205_preview_item_in_portal?v=2&t=PRODUCT_DETAILS

**Comment by vedang444** [comment]
You can use the table name as well in the shortcut that you have created.
/sp?id=**$table**&sys_id=$sysid

This will ensure the correct one opens up, eg. sc_cat_item_guide if it is Order Guide and sc_cat_item if it is a Catalog Item.
I think doesn't work for record producer but you can always have a 2nd slashcommand for now.

**Reply by krysbzh** [reply]
Sometimes, I want to try the catalog item in different portals (/sp , /esc...) . So I added a parameter to choose the portal prefix in my command :
Command : try
URL : /$0?id=$table&sys_id=$sysid
Hint : Open Cat Item/order guide in portals
